### DISCUSSION TITLE
Facing issue while login to github

### DISCUSSION THREAD

**Discussion by rajan0917** [community/community#54335] [New to GitHub]
Hi Team

            I have rejoin to XXXXX company after 1 year, 

When i am trying to sing up then its say mail id already exist .

and 

When i am trying to reset my password then its asking for 6 digit security (MFA)  code or recovery code.

My 2 factor is not working (might be that is old ) and i have lost my security code (Old laptop data format)

Could any one help me to fix this issue? so that i will login to my account.

**Comment by airtower-luna** [comment]
Without 2FA, recovery code, or one of the other supported recovery methods you won't be able to get the account back, the [documentation is pretty clear on the matter](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). What you *can* do is disconnect the email address from the old account, and make a new account with it (or add it to your current one), see: [Unlinking your email address from a locked account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account)

**Reply by rajan0917** [reply]
Thanks for help

unfortunately i am not able to login as i join my organization after one year with same mail id

Actually i when ever i tried to reset my password then it only as for 6 digit authentication code or a recovery code

**Reply by airtower-luna** [reply]
I understood that. If you don't have those you won't be able to recover the account.

But you can remove the *email address* from the old account, and then make a new one with that address, or just add the address to the account you're using right now. Check the documentation links I posted above.
